### PR TITLE
Add transform for MasterConfig.ProjectConfig

### PR DESCRIPTION
**Pull request by gildub**

### CODE REVIEW

File: pkg/transform/project_transform.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+			Comment:    "Not supported in OCP4: " + strconv.Itoa(e.ProjectConfig.SecurityAllocator.MCSLabelsPerProject),
+		})
+
+	reportOutput.Reports = append(reportOutput.Reports,
+		Report{
+			Name:       "SecurityAllocator.uidAllocatorRange",
+			Kind:       "ProjectConfig",
+			Supported:  false,
+			Confidence: NoConfidence,
+			Comment:    "Not supported in OCP4: " + string(e.ProjectConfig.SecurityAllocator.UIDAllocatorRange),
```
Comment (alexander-demichev) [inline pkg/transform/project_transform.go]: Golang way of concatenating strings is `sprintf`, but that also looks ok for me after working with `Ruby`

File: pkg/transform/project/project.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	if projectConfig.ProjectRequestTemplate != "" {
+		projectCR.Spec.ProjectRequestTemplate.Name = projectConfig.ProjectRequestTemplate
+	}
+
+	return &projectCR, nil
+}
+
+// Validate registry data collected from an OCP3 cluster
+func Validate(e legacyconfigv1.ProjectConfig) error {
+	return nil
```
Comment (alexander-demichev) [inline pkg/transform/project/project.go]: Is there any reason to skip validation?

File: pkg/transform/project_transform.go (hunk 1 above)
Comment (gildub) [reply pkg/transform/project_transform.go]: I remember reading about such practice, but you're right `Ruby` habit has taken over!
I've changed it. Thank you.

File: pkg/transform/project/project.go (hunk 2 above)
Comment (gildub) [reply pkg/transform/project/project.go]: All the fields are optional so there is nothing much to enforce.

**Pull request merged**